src/js/utils.js

```diff
@@ -21,6 +21,7 @@ exports.parseQueryParams = () => {
 /**
  * Creates an object based on a form's current input values.
  *
+ * @param {object} form
```

Comment:
This is a minor cleanup spotted by the newer eslint-jsdoc plugin, which was bumped on `master` earlier today